Website/src/rendering/src/assets/css/components/PageContent/PreviewSearch.css

```diff
@@ -0,0 +1,100 @@
+.preview-search {
+  @apply absolute w-screen top-8 left-0;
+  @apply lg:top-14;
+
+  @media (min-width: 1024px) {
+    left: -148px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .preview-search-modal-container {
+    min-height: 300px;
+    max-width: 1440px;
+
+    @apply relative bg-gray-light w-full flex flex-row flex-wrap border border-solid border-gray pt-4;
+    @apply lg:flex-initial;
+
+    .left-section {
+      @apply w-full mb-4;
+      @apply lg:w-1/5;
+
+      .list-container {
+        @apply w-full;
+
+        .list-container-title {
+          @apply font-bold;
+        }
+
+        .list-item {
+          @apply h-6;
+
+          a {
+            @apply text-blue;
+          }
+        }
+      }
+    }
+
+    .right-section {
+      @apply w-full;
+      @apply lg:w-4/5;
+
+      .right-section-title {
+        @apply mb-3 font-bold;
+        @apply lg:mb-1;
+      }
+
+      .view-all-btn {
+        @apply absolute right-5 top-32 text-blue font-medium;
+        @apply lg:top-4;
+      }
+
+      .product-list-container {
+        @apply flex flex-wrap justify-between;
+
+        .product-list-item {
+          width: 49%;
+
+          @media (min-width: 1024px) {
+            width: 32%;
+          }
+
+          @apply h-60 text-center mb-3;
+          @apply sm:h-96;
+          @apply lg:h-80;
+
+          .product-container {
+            @apply w-full h-full inline-block text-center relative align-top bg-transparent border border-solid border-transparent;
+
+            .product-image-container {
+              @apply relative h-2/3 w-full inline-block border border-solid border-gray;
+
+              .product-image {
+                @apply relative inline-block max-w-full max-h-full;
+              }
+            }
+
+            .product-info-container {
+              @apply bg-white h-1/3 pt-2;
+
+              .product-name {
+                @apply font-bold border-0 overflow-hidden text-black leading-5 text-left px-1 h-10;
```

Comment:
@chpa7 
The design has ".." on the product names and brands to indicate the text should be overflow hidden with ellipsis if too long. They should be single line. Could you implement that instead of `h-10`?